httomo/globals.py

```diff
@@ -7,7 +7,7 @@
 MAX_CPU_SLICES: int = (
     64  # A some random number which will be overwritten by --max-cpu_slices flag during runtime
 )
-FRAMES_PER_CHUNK: int = 1  # if given as 0, then write contiguous (no chunking)
+FRAMES_PER_CHUNK: int = 1  # if given as 0, then write contiguous (no chunking); if given as -1, decide automatically
```

Comment:
Do I understand it correctly, there is 3 cases here:
1. 0 write contiguous (no chunking)
2. > 0 set to a desired number of chunks per frame explicitly
3. -1 automatic selection of chunks

Reply:
Yes there are 3 cases like what you described. I am just not sure overloading `frames-per-chunk` in this way is how httomo does thing.